src/python/k4a/docs/building.md

```diff
@@ -0,0 +1,123 @@
+# Building Azure Kinect Python API(K4A)
+
+This information is for how to build your own copy of the Python wheel. 
+If you need to build from source, you have to clone the repository to ensure all the submodule dependencies in place.
+
+## Support Configurations
+
+Python:
+* Greater than or equal to version 3.6.
+
+Architectures: 
+* amd64
+* x86 (limited, no testing is performed)
+
+OS:
+* Windows
+* Linux
+
+## Dependencies
+
+Building on Windows and Linux requires an active internet connection. Part of the
+build will require downloading dependencies.
+
+The following dependencies are for both Windows and Linux:
+
+* [python3](https://www.python.org/getit/). During the install make sure to add
+  python to path.
```

Comment:
Also need venv

Reply:
as well as pip and doxygen, but on my system I only needed to manually install venv additionally

Reply:
Please fix.

Reply:
I did notice that venv is not automatically installed as a Python package in Linux, even though venv is part of the Python core modules. It is part of Python on Windows...In any case, I added a note to install venv in Linux.

There is a comment that doxygen is an optional tool.

As for pip, it should really be installed as part of Python. It is the official Python package installer, so yeah. If it is not part of Python installation, then nothing will work. In any case, I added a comment about it.